File: 1929-concatenation-of-array/1929-concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] result = new int[2*nums.length];
        for(int i=0;i<=nums.length-1;i++){
            result [i] = nums[i];
            result[i+nums.length]= nums[i];
        }
        return result;
    }
}

File: 1929-concatenation-of-array/1929-concatenation-of-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    const arr = [];
    for(let i=0;i<nums.length;i++){
        arr.push(nums[i]);
    }
    for(let i=0;i<nums.length;i++){
        arr.push(nums[i]);
    }
    return arr;
};